administrator/components/com_media/resources/styles/components/_layout.scss

```diff
@@ -34,7 +34,6 @@
 .media-main {
   position: relative;
   height: calc(100vh - 160px);
```

Comment:
It still does not work, but when I remove this line then it should be ok.

Reply:
Ok. I see now. The issue is only present in debug mode.

Removing this line means the grid/list area does not scroll. All images are displayed and instead you use the default browser scroll. Originally  the idea to adding the scroll to the grid/list area was so you can scroll the images while still keeping the toolbar and sidebar displayed. Should I still removed it?

Reply:
You are the CSS master. I do just report what I see.